README.md

```diff
@@ -54,18 +67,21 @@ runtime, respectively.
    own module.
 1. Prepare connector credentials and/or other parameters.
     - In order to pass parameters to your connector's instances, you have to create a property file.
-    - In the `plugin-tester` directory create a Java property file named `<connector-id>.properties`, 
+    - In the `plugin-tester` directory create a Java property file named `<connector-id>.properties`,
       e.g. for the Everytrade Kraken Connector, the file name is *everytrade.krkApiConnector.properties* bacause the
       connector's ID is *krkApiConnector* and the connector parent plugin's ID is *everytrade*.
-   - The file's content will be loaded into a map as key-value pairs and used to initialize the connector's 
+   - The file's content will be loaded into a map as key-value pairs and used to initialize the connector's
      instance. Your code can read the data in the `createConnectorInstance` method implementation. The template
      implementation just passes the data into the connector's constructor.
+1. Prepare test CSV files with transactions which you want to parse in the `parser-files` directory.
 1. Run the Gradle task `:plugin-tester:run` either via your IDE of choice (recommended, as you can debug your plugin
    this way) or via the command line (in the repository's root dir, run `./gradlew :plugin-tester:run`).
-1. Watch the tester's output to determine whether correct data is downloaded.
-1. The tester specifically tries two passes of the download procedure in order to check whether your connector can
-   correctly resume download from where it left off after the first download pass. In order to fully test this, you
-   should adjust your connector when testing to limit the amound of data being downloaded in a single pass.     
+1. Watch the tester's output to determine whether correct data is downloaded and correct data is parsed.
+    - For the tested connector the tester specifically tries two passes of the download procedure in order to check
+     whether your connector can correctly resume download from where it left off after the first download pass. In
+      order to fully test this, you should adjust your connector when testing to limit the amound of data being
+      downloaded in a single pass.
+   - The tester try to find and parse each file in the `parser-files` directory with appropriate parser.
```

Comment:
The tester **tries to parse** parse each file ... with **an** appropriate parser. **A parser is chosen based on a match between the CVS file's header and one of the parser's advertised headers**.